### PR TITLE
Personal/steve/data mapping

### PR DESCRIPTION
**Pull request by SteveWohl**
Hi Ranvijay and Quan,

The edits to the Data-Mapping article are complete.

I was a little confused by the numeric headings. I removed the numbers from the headings, mainly because, they are not recommended from an Azure Docs perspective. I realize this is open-source content.  Therefore, please review my changes and reach out to me if you have any questions or concerns about them.  I tried not to rewrite much in the headings and in the paragraphs. I did basic copyedits for grammar and spelling. 

Regards,
Steve

### CODE REVIEW

File: FhirToDataLake/docs/Data-Mapping.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
+# Data mapping from FHIR to Synapse
+The Synapse sync pipeline fetches, converts, and saves FHIR data to the Azure storage in the [Apache Parquet format](https://docs.microsoft.com/en-us/azure/databricks/data/data-sources/read-parquet). The Apache Parquet format is an open source file format designed for efficient and performant flat columnar storage format of data compared to row based files like CSV or TSV files. For more information about its detailed storage strategy, see the [Apache Parquet](https://parquet.apache.org/documentation/latest/) documentation.
+
+Azure Synapse Analytics supports [Parquet files](https://docs.microsoft.com/en-us/azure/synapse-analytics/sql/query-parquet-files) in Serverless SQL pool. PowerShell [Set-SynapseEnvironment.ps1](../scripts/Set-SynapseEnvironment.ps1) is provided to users to create default EXTERNAL TABLEs/VIEWs on Synapse. Refer to the default [SQL scripts](https://github.com/microsoft/FHIR-Analytics-Pipelines/tree/main/FhirToDataLake/scripts/sql/Resources) for information about creating customized EXTERNAL TABLEs or VIEWs.
```
[the remark this answers is not in the document]
Comment (QuanWanxx) [reply FhirToDataLake/docs/Data-Mapping.md]: I thought about it. Customers may edit or even try SQL scripts, that may cause the query to fail and is very difficult to track, so I updated here, don't give obvious reference for the SQL scripts, give Azure Synapse EXTERNAL TABLEs and VIEWs documents.

File: FhirToDataLake/docs/Data-Mapping.md [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+
+The information below descibes how to convert raw FHIR JSON data to Parquet format and how to define the default EXTERNAL TABLEs/VIEWs.
```
[the remark this answers is not in the document]
Comment (QuanWanxx) [reply FhirToDataLake/docs/Data-Mapping.md]: Updated

File: FhirToDataLake/docs/Data-Mapping.md [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+
+## FHIR JSON data to Parquet data
+
+Parquet format supports nested and repeated structure. Theoretically, JSON data can be a lossless convert to Parquet format, but for stability and practical purposes, we designed piece schemas to transform FHIR raw JSON data into Parquet following the rules listed below. The detailed schemas are available in the [**data/schemas**](https://github.com/microsoft/FHIR-Analytics-Pipelines/tree/main/FhirToDataLake/data/schemas) directory.
```
[the remark this answers is not in the document]
Comment (QuanWanxx) [reply FhirToDataLake/docs/Data-Mapping.md]: Updated.

File: FhirToDataLake/docs/Data-Mapping.md [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+
+Data saved in Parquet format is human unreadable due to the compressed and storage strategy. To view a sample Python script read a Parquet file with specific columns for a patient resource, see  [Using Python library to quickly read Parquet files](#using-python-library-to-quickly-read-parquet-files).
```
[the remark this answers is not in the document]
Comment (QuanWanxx) [reply FhirToDataLake/docs/Data-Mapping.md]: thanks, updated

**Review by SteveWohl — COMMENTED** [review]
I edited the sentence in line 12.

**Review by QuanWanxx — APPROVED** [review]
Maybe we can update the main page, add reference for this document as offline emails discussed.

**Comment by SteveWohl** [comment]
@ranvijaykumar @QuanWanxx 

I am signing this PR off per your request in the comments of this PR and in the e-mail message sent to me. I'll update the second paragraph of the Deployment.md file also called “FHIR to Synapse Sync Agent” that will link to the Data-Mapping.md file also called "Data mapping from FHIR to Synapse" that's specific to this PR.

**Pull request merged**